centreon/www/class/centreon-clapi/centreonAPI.class.php

```
@@ -1322,6 +1212,116 @@ public function sortClassExport(): void
+                        . '.class.php';
+                }
+            }
+
+            if (
+                isset($this->relationObject[$object]['libs'])
+                && ! array_walk($this->relationObject[$object]['libs'], 'class_exists')
+            ) {
+                array_walk($this->relationObject[$object]['libs'], 'require_once');
+            }
+        } else {
+            foreach ($this->relationObject as $sSynonyme => $oObjet) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sSynonyme'. (Unused Code Rules)

(UnusedLocalVariable)